lib/xnvme_be_macos_driverkit_admin.cpp

```diff
@@ -0,0 +1,63 @@
+// SPDX-FileCopyrightText: Samsung Electronics Co., Ltd
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <map>
```

Comment:
It's needed for `#include <xnvme_be_macos_driverkit.h>`

Reply:
In that case, shouldn't it be inside `xnvme_be_macos_driverkit.h` instead?

Reply:
It's tricky, because it cannot be included from within `extern "C" {`, as it's a C++ dependency. And `xnvme_be_macos_driverkit.h` is included from both cpp and c files.

I've made a workaround now.